web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js

```diff
@@ -123,19 +123,38 @@ odoo.define('web_advanced_search_x2x.search_filters', function (require) {
                 event.stopPropagation();
             });
         },
+        is_boolean_operator: function(element){
```

Comment:
@lmignon @benwillig is there any utils method in Odoo that does what I'm trying to do here with my method? if yes can you please point it to me